client/blocks/reader-post-card/blocked.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
```

Comment:
Thanks @obenland. I'm now using it for `bumpStat` and `recordGoogleEvent` (aed56e5).

I've continued using `recordTrack` from reader/stats for now, because we add some extra metadata there:

https://github.com/Automattic/wp-calypso/blob/master/client/reader/stats.js#L90